app/src/main/java/io/apicurio/registry/ccompat/dto/CompatibilityLevelDto.java

```diff
@@ -41,6 +42,7 @@
 @Getter
 @EqualsAndHashCode
 @ToString
+@RegisterForReflection
```

Comment:
Remind me why these annotations are needed?

Reply:
yes, exactly, there are some classes that quarkus is smart enough to detect and automatically register them, but some others like when we manually SerDe them using `ObjectMapper` not, so we have to add this annotation

Reply:
yes, exactly, there are some classes that quarkus is smart enough to detect and automatically register them, but some others like when we manually SerDe them using `ObjectMapper` not, so we have to add this annotation